renderlib/graphics/RenderGL.cpp

```diff
@@ -100,11 +103,60 @@ RenderGL::renderTo(const CCamera& camera, GLFramebufferObject* fbo)
 
   doClear();
   if (haveScene) {
-    m_image3d->render(camera, m_scene, m_renderSettings);
+    drawSceneObjects(camera);
   }
   fbo->release();
 }
 
+void
+RenderGL::drawSceneObjects(const CCamera& camera)
```

Comment:
most of the lines of code in here are just getting the bounding box's transform matrix to position and scale it correctly in the world with the volume being drawn.  This is repeated code from the pathtrace renderer and can be DRY refactored at some point.